src/app/widgets/Rotary/StepsToggle.js

```diff
@@ -3,34 +3,64 @@ import React, { useState } from 'react';
 import store from 'app/store';
 import ToggleSwitch from 'app/components/ToggleSwitch';
 import Tooltip from 'app/components/TooltipCustom/ToolTip';
+import ToolModal from 'app/components/ToolModal/ToolModal';
+import Button from 'app/components/FunctionButton/FunctionButton';
+import reduxStore from 'app/store/redux';
+import get from 'lodash/get';
 import { ROTARY_AXIS_101_VALUE, WORKSPACE_MODE } from 'app/constants';
 import { updateWorkspaceMode } from 'app/lib/rotary';
+import styles from './index.styl';
 
 const { DEFAULT, ROTARY } = WORKSPACE_MODE;
 
 const StepsToggle = () => {
     const [workspaceMode, setWorkspaceMode] = useState(store.get('workspace.mode', DEFAULT));
-
+    // eslint-disable-next-line no-unused-vars
+    const [wiringNotification, setWiringNotification] = useState(false);
+    const firmwareType = get(reduxStore.getState(), 'controller.type');
```

Comment:
Same drill, since this is a functional component you can use the `useSelector` hook to access the redux state easily